test/test_loader.py

```diff
@@ -51,6 +51,21 @@ def example_url_lazy(bot, trigger):
     pass
 
 
+@sopel.plugin.rule_lazy(loader)
+def example_rule_lazy(bot, trigger):
+    pass
+
+
+@sopel.plugin.find_lazy(loader)
+def example_find_lazy(bot, trigger):
+    pass
+
+
+@sopel.plugin.search_lazy(loader)
+def example_search_lazy(bot, trigger):
+    pass
+
+
```

Comment:
I wouldn't mind at all having a separate commit that adds normal rule/find/search to this test suite.

Also understandable if you want to do that separately, but it's certainly weird that we don't already have those.

Reply:
Oh, you know, our test suite isn't perfect yet. 😬